plugins/lookup/aws_secret.py

```diff
@@ -31,6 +31,11 @@
   version_stage:
     description: Stage of the secret version.
     required: False
+  nested:
+    description: Indicates the secret contains nested values. The secret can be queried using the following syntax: `aws_secret_object_name.key1.key2.key3`. If an object is of the form `{"key1":{"key2":{"key3":1}}}` the query would return the value `1`.
+    type: boolean
+    default: False
+    required: False
```

Comment:
One of the error is caused by the ‘:’ character used into the description. May you move this part **“The secret can be queried using the following syntax: `aws_secret_object_name.key1.key2.key3`. If an object is of the form `{“key1”:{“key2":{“key3”:1}}}` the query would return the value `1`.“** into a module usage example and explaining the returning values as well?

Could can remove the **"required: False"** as it is optional here.

Could you add the **version_added** field?

Reply:
I've moved the description under the module usage examples, removed the required field, and added the version_added field (as 1.4.0) in 243e4e00f. I had to rebase my branch to latest main due to merge conflicts.